locales/en/app.ftl

```diff
@@ -8,6 +8,7 @@
 -product-short-name = Monitor
 -brand-name = Firefox
 -brand-Mozilla = Mozilla
+-brand-premium = Premium
```

Comment:
I don't think we should define premium as a brand.

In case, it shouldn't go at the beginning of this file. See line 652 and https://mozilla-hub.atlassian.net/browse/MNTOR-1562